wallet/src/de/schildbach/wallet/ui/username/voting/RequestUserNameViewModel.kt

```diff
@@ -247,7 +253,7 @@ class RequestUserNameViewModel @Inject constructor(
     }
 
     fun checkUsername(requestedUserName: String?) {
-        viewModelScope.launch {
+        viewModelScope.launch(Dispatchers.IO) {
```

Comment:
In general, it's better not to dispatch coroutines in IO scope at the top level, but go down to where the actual work is done and call it inside `withContext(Dispatchers.IO)` so that the top-level callers don't have to worry about threading, correct context and switching back to the main thread to update UI.

Reply:
thanks for this idea.

Reply:
I will remove this `(Dispatchers.IO)` and use `withContext` in the next PR